shell/browser/ui/cocoa/electron_ns_window.mm

```diff
@@ -203,25 +203,23 @@ - (void)sendEvent:(NSEvent*)event {
       (event.type == NSEventTypeLeftMouseDown &&
        ([event modifierFlags] & NSEventModifierFlagControl))) {
     // The WebContentsView is added a sibling of BaseWindow's contentView at
-    // index 0 before it in the paint order - see
-    // https://github.com/electron/electron/pull/41256.
+    // index 0 before it in the paint order in BrowserWindows.
     const auto& children = shell_->GetContentsView()->children();
-    if (children.empty())
-      return;
-
-    auto* wcv = children.front().get();
-    if (!wcv)
-      return;
-
-    auto ns_view = static_cast<electron::DelayedNativeViewHost*>(wcv)
-                       ->GetNativeView()
-                       .GetNativeNSView();
-    if (!ns_view)
-      return;
-
-    [static_cast<ElectronInspectableWebContentsView*>(ns_view)
-        redispatchContextMenuEvent:base::apple::OwnedNSEvent(event)];
-    return;
+    // Could be a BaseWindow or a BrowserWindow - the WebContentsView is
+    // only added to BrowserWindow - see
+    // https://github.com/electron/electron/pull/41256.
+    if (children.size() >= 2) {
```

Comment:
This line is the main change, if I'm reading the PR right,  but even with the new comment I don't really understand the context enough to agree or disagree with it. @codebytere can you walk me through the logic?

Reply:
@ckerr updated and clarified - let me know if that's enough!